templates/repo/diff/box.tmpl

```diff
@@ -1,5 +1,5 @@
 {{if .DiffNotAvailable}}
-	<div class="diff-detail-box diff-box ui sticky">
+	<div class="diff-detail-box diff-box ui">
 		<div>
```

Comment:
I'd move this sticky to this bare `<div>`. The intention clearly seems to be that the line with the review button is always in sight, like on GitHub.